modules/text/src/text_detector_swt.cpp

```diff
@@ -92,14 +93,25 @@ static
 void DFSUtil(int v, std::vector<bool> & visited, std::vector< std::vector<int> >& adj, int label, std::vector<int> &component_id)
 {
     // Mark the current node as visited and label it as belonging to the current component
```

Comment:
This comment is related to removed/moved code.

Reply:
moved the comments